health-services/project-factory/src/server/processFlowClasses/user-processClass.ts

```
@@ -0,0 +1,225 @@
+            const map: any = {};
+            for (const user of response?.Employees) {
+                map[user?.user?.mobileNumber] = user?.user?.userServiceUuid;
+            }
+            return map;
+        } catch (error: any) {
+            console.error("Employee creation API failed:", error);
+            throw new Error(error);
+        }
+    }
+
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)